config/infection.json.dist

```diff
@@ -58,6 +60,7 @@
         },
         "Throw_": {
             "ignore": [
+                "Psl\\DateTime\\DateTime::__construct",
```

Comment:
-> It's impossible to test the InvalidArgumentExceptions that are being thrown. I'm wondering if it would make more sense to use the same checks from `fromParts` in the constructor directly.

The constructor does a range check, the fromParts does a calendar (intl) check. 
Both result in different type of exceptions with different messages.

Would it make sense to always go for the fromParts() checks instead?